docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -11,56 +11,61 @@ tags:
 
-The following is an example List Filter:
+### Supported operators
 
-```
-WorkflowType = "main.YourWorkflowDefinition" and ExecutionStatus != "Running" and (StartTime > "2021-06-07T16:46:34.236-08:00" or CloseTime > "2021-06-07T16:46:34-08:00") order by StartTime desc
-```
+:::note
+
+Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+:::
+
+A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and the following supported operators:
 
-[More example List Filters](#example-list-filters)
+- **=, !=, >, >=, <, <=**
+- **AND, OR, ()**
+- **BETWEEN ... AND**
+- **IN**
+- **ORDER BY**
 
-A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and Operators.
+The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
 
-- The following operators are supported in List Filters:
+### Partial string match
 
-  - **=, !=, >, >=, <, <=**
-  - **AND, OR, ()**
-  - **BETWEEN ... AND**
-  - **IN**
-  - **ORDER BY**
-  - **LIKE**
+The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
+<!-- note: advanced vis features will be supported in SQL upon the release of v1.20.-->
 
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+For example, if you have a Search Attribute `Description` of `Text` type with the value of "The quick brown fox jumps over the lazy dog", searching for `Description=quick` or `Description=fox` will successfully return the Workflow.
+However, partial word searches such as `Description=qui` or `Description=laz` will not return the Workflow.
+This is because [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
 
-- A List Filter applies to a single Namespace.
+### Efficient API usage
 
-- The range of a List Filter timestamp (StartTime, CloseTime, ExecutionTime) cannot exceed 9223372036854775807 (that is, maxInt64 - 1001).
+An Advanced List Filter API may take longer to respond if it is retrieving a large number of Workflow Executions (over 10,000).
 
-- A List Filter that uses a time range has a resolution of 1 ms on Elasticsearch 6 and 1 ns on Elasticsearch 7.
+<!-- Use the `CountWorkflow` API to efficiently count the number of [Workflow Executions](/concepts/what-is-a-workflow-execution). -->
```

Comment:
We have delayed v1.20 release, and we were able to include `CountWorkflow` in it. This can be uncommented now.

Reply:
Until 1.2 is out, this is staying commented.

Reply:
We can make another update in the future for all 1.2 updates. This PR was focused on correcting existing information that could mislead users at present.